src/main/kotlin/xyz/atrius/waystones/commands/InfoCommand.kt

```diff
@@ -8,10 +8,13 @@ object InfoCommand : SimpleCommand("info", "i") {
 
     override fun execute(sender: CommandSender, args: Array<String>) {
         sender.message("""
-            |------------ &dWaystones&r ------------
-            |Version: &a${plugin.description.version}&r
-            |Author: &dAtri&r
-            |-----------------------------------
+            |
+            |&7------------- &dWaystones &7------------&r
+            |
+            |Version&7: &a${plugin.description.version}&r
+            |Author&7: &dAtri&r
+            |
+            |&7---------------&b-&d-&f-&d-&b-&7---------------
         """.trimMargin())
```

Comment:
I think we should change this to reference the list of contributors instead of hardcoding the author.
```suggestion
		val author = plugin.description.authors.getOrNull(0) ?: "Unknown"
        val contributors = plugin.description.authors.drop(1)
            .joinToString(", ") { "&d$it&r" }
        // Print out plugin info
        sender.message("""
            |&7------------- &dWaystones &7------------&r
            |
            |Version: &a${plugin.description.version}&r
            |Author: $author
            |Contributors: $contributors
            |
            |&7---------------&b-&d-&f-&d-&b-&7---------------
        """.trimMargin())
```

Reply:
In addition to this maybe it should also include references to the website too via `plugin.description.website`

Reply:
If anything it might actually be better to store the strings outside the method so they don't need to be pulled and processed repeatedly.

Reply:
> In addition to this maybe it should also include references to the website too via `plugin.description.website`

This is something I was considering, a link to this repo, but if I'm not mistaken that'll take some reworking as embedding links relies on the [`tellraw` command](https://minecraft.gamepedia.com/Commands#tellraw)

> might actually be better to store the string outside the method

> I think we should change this to reference the list of contributors instead of hardcoding the author.

Both are fair points, sadly I won't be able to get to these two until a bit later tonight.

Reply:
I don't know if you need to use `tellraw` for links, generally the game is smart enough to parse them on their own isn't it?